recipes/default.rb

```diff
@@ -21,7 +21,7 @@
 
 # Start+Enable the lvmetad service on RHEL7, it is required by default
 # but not automatically started
-if node['platform_family'] == 'rhel' && node['platform_version'].to_i >= 7
+if node['platform_family'] == 'rhel' && node[platform] != 'amazon' && node['platform_version'].to_i >= 7
```

Comment:
It will be totally enough.
If chef-client 13+, then:
   node['platform_family'] == 'rhel'   will be false.
else if chef-client <=12, then:
  node[platform] != 'amazon'   will be false.